source/wp-content/themes/wporg-developer-2023/inc/template-tags.php

```diff
@@ -885,7 +885,7 @@ function get_param_reference( $param, $recursion_limit = 3 ) {
 					'posts_per_page' => 1,
 				);
 			
-				$query = new WP_Query( $args );
+				$query = new \WP_Query( $args );
```

Comment:
I think this might be the only one that's actually needed, as this function is within the DevHub namespace